content/qchem/heh-cation/meta.json

```
@@ -30,6 +34,6 @@
   "changelog": [
     "version 0.1 : initial public release"
   ],
```

We can try something like this with a few more details:
```suggestion
  "changelog": [
    "version 0.2: added initial states",
    "version 0.1 : initial public release"
  ],
```

If it doesn't work, maybe it should all be one string.

---

Awesome, will do